src/client/api/init.c

```diff
@@ -212,9 +214,23 @@ daos_init(void)
 	if (rc != 0)
 		D_GOTO(out_co, rc);
 
+
+	// Initializing for the client just to play with it
+	// There's an open question how we want the API functions to behave
+	// in the shared client/server code if the client doesn't initialized
+	// the API.  They currently will fail, and generate errors.  Might need
+	// to quiet the errors completely to avoid log spam.  Or somehow
```

Comment:
```suggestion
	/* Initializing for the client just to play with it */
	/* There's an open question how we want the API functions to behave */
	/* in the shared client/server code if the client doesn't initialized */
	/* the API.  They currently will fail, and generate errors.  Might need */
	/* to quiet the errors completely to avoid log spam.  Or somehow */
	/* conditionally compile the metrics for the server, if possible. */
```